VMT/templates/reception-embargo-reminder.md

```diff
@@ -0,0 +1,16 @@
+# Reception embargo reminder (private issues) template
+
+```
+This issue is being treated as a potential security risk under embargo.
```

Comment:
The important takeaway here is just to make sure that everyone looking at an embargoed report of a suspected vulnerability is reminded it should be discussed only within the confines of the private defect tracker or via consent of those coordinating its disclosure, to avoid premature leakage of details. This includes not pushing patches into a public Git repository/code review system/source tarball/et cetera, and not deploying fixes with publicly-visible changes in behavior in advance of the scheduled disclosure.

These are all things we've seen happen in the past, so over time we adjusted the wording to be increasingly comprehensive, but I agree it might be verging on incomprehensible at this stage.